controllers/masteruserrecord/masteruserrecord_controller.go

```diff
@@ -144,32 +142,23 @@ func (r *Reconciler) ensureUserAccount(logger logr.Logger, murAccount toolchainv
 			// does not exist - should create
 			userAccount = newUserAccount(nsdName, murAccount.Spec, mur.Spec)
 			if err := memberCluster.Client.Create(context.TODO(), userAccount); err != nil {
-				return 0, r.wrapErrorWithStatusUpdate(logger, mur, r.setStatusFailed(toolchainv1alpha1.MasterUserRecordUnableToCreateUserAccountReason), err,
+				return r.wrapErrorWithStatusUpdate(logger, mur, r.setStatusFailed(toolchainv1alpha1.MasterUserRecordUnableToCreateUserAccountReason), err,
 					"failed to create UserAccount in the member cluster '%s'", murAccount.TargetCluster)
 			}
-			counter.IncrementUserAccountCount(logger, murAccount.TargetCluster)
-			return 0, updateStatusConditions(logger, r.Client, mur, toBeNotReady(toolchainv1alpha1.MasterUserRecordProvisioningReason, ""))
+			if murAccount.SyncIndex != "0" && murAccount.SyncIndex != "deleted" {
```

Comment:
Should we really check if it's not "0"? It's "0" for the case when it's just created (but not synced yet), right? But we still need to count it, no?

Reply:
if the MUR is created with the UserAccount def in its spec, then it contains the default value of the `syncIndex` - don't forget that it's a string, so the default value will be an empty string: `""`
We don't set anything specific to the syncIndex when creating MUR: https://github.com/codeready-toolchain/host-operator/blob/master/controllers/usersignup/masteruserrecord.go#L52-L62
I added the condition, because right now the useraccount_status controller sets the syncIndex to zero, when the UserAccount is gone. As soon as I modify it to `delete` then I can drop the zero from this condition. But still, it shouldn't have any negative effect at all

Reply:
Ah right. Forgot that it's a string. So the default value is "". Not 0.